### PR TITLE
Add image sync information

### PR DESCRIPTION
**Pull request by aaannz**
Image synchronization is required part of installing new branch server and also day-to-day operations. This PR adds this info in various relevant places.

### CODE REVIEW

**Comment by aaannz** [comment]
@Loquacity As we were discussing today, this is part of the #235 which I think is necessary to have.

File: adoc/retail_chap_admin.adoc [hunk 1]
```diff
@@ -41,6 +41,7 @@ This configuration allows for the branch server to provide DHCP, DNS, TFTP, PXE
 * Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
 * Specify the working directory, and the directory owner and group.
 . Click btn:[Save] to save your changes.
+. Apply the hightstate.
```
Comment (Loquacity) [inline adoc/retail_chap_admin.adoc]: highstate

File: adoc/retail_chap_admin.adoc [hunk 2]
```diff
@@ -70,11 +71,7 @@ Ensure you include DNS, TFTP and FTP services.
 * Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
 * Specify the working directory, and the directory owner and group.
 . Click btn:[Save] to save your changes.
-. Apply the salt state and synchronize your settings:
-+
-----
-salt $branch_server_id state.apply image-sync
-----
+. Apply the hightstate.
```
Comment (Loquacity) [inline adoc/retail_chap_admin.adoc]: highstate

File: adoc/retail_chap_install.adoc [hunk 3]
```diff
@@ -238,21 +238,33 @@ Do not install the branch server as a traditionally managed client.
...
 === Assign and configure branch server formulas
 
-Before you configure the branch server formulas, ensure you have configured networking, and know the Salt ID of the branch server.
+Before you configure the branch server, ensure you have decided on networking topology, and know the Salt ID of the branch server.
 
 .Procedure: Configuring branch server formulas
 
 . Branch server configuration is performed using the [command]``retail_branch_init`` command:
 +
+.Configure branch server using default values and single shared network interface
```
Comment (Loquacity) [inline adoc/retail_chap_install.adoc]: Needs a space between the dot and the step content.

File: adoc/retail_chap_install.adoc [hunk 4]
```diff
@@ -238,21 +238,33 @@ Do not install the branch server as a traditionally managed client.
...
 ----
 retail_branch_init <branch_server_salt_id>
 ----
 +
 If you need to specify values for your architecture, use:
 +
+.Configure branch server using specified values and dedicated network interface for branch network
```
Comment (Loquacity) [inline adoc/retail_chap_install.adoc]: Needs a space between the dot and the step content.

File: adoc/retail_chap_install.adoc (hunk 3 above)
Comment (aaannz) [reply adoc/retail_chap_install.adoc]: I do not mean it as a step content, but as a description of the example.
I want to show two different ways to call `retail_branch_init` script

Comment (Loquacity) [reply adoc/retail_chap_install.adoc]: In that case, we probably need to review the procedure. Can you remove the example from within the procedure, and put it separately at the end? You don't want to mix up content types.

**Pull request merged**